### PR TITLE
Add staked to other accounts value to balances

### PR DESCRIPTION
**Pull request by dafuga**
Small PR for https://github.com/greymass/eos-voter/issues/335

### CODE REVIEW

File: app/renderer/assets/locales/de-DE/wallet.json [hunk 1]
```diff
@@ -94,4 +94,4 @@
   "wallet_status_tab_staked": "Staked EOS",
   "wallet_status_tab_token_balances": "Balances",
   "wallet_status_total_balance": "Total"
-}
\ No newline at end of file
+}
```
Comment (aaroncox) [inline app/renderer/assets/locales/de-DE/wallet.json]: Can you revert this change? This is pointless and is going to end up causing conflicts with crowdin (I think). The non-english files shouldn't be altered at all.

Comment (dafuga) [reply app/renderer/assets/locales/de-DE/wallet.json]: not sure how that change got in there lol, but I reverted it! 😄
